packages/react/src/primitives/types/breadcrumbs.ts

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Sizes } from './base';
+import { ElementType, PrimitiveProps, BaseViewProps } from './view';
+import { BaseLinkProps } from './link';
+
+/** @deprecated For internal use only */
+export interface BaseBreadcrumbProps extends BaseViewProps {
```

Comment:
Out of scope for this PR, but what if we stop exporting all our primitive types so we no longer need to add deprecation messages for new types?

Reply:
I like that, it feels weird to use these deprecated comments